firmware/application/ui_navigation.cpp

```diff
@@ -330,21 +332,10 @@ SystemStatusView::SystemStatusView(
     toggle_fake_brightness.on_change = [this, &nav](bool v) {
         set_dirty();
         pmem::set_apply_fake_brightness(v);
-        if (nav.is_valid() && v) {
-            nav.display_modal(
-                "Brightness",
-                "You have enabled brightness\n"
-                "adjustment. Performance\n"
-                "will be impacted slightly.");
-
-            // TODO: refresh interface to prevent reboot requirement
-            // TODO: increase performance
-        } else if (!v) {
-            nav.display_modal(
-                "Brightness",
-                "Brightness adjust disabled.");
-        }
         refresh();
+        if (nullptr != parent()) {
```

Comment:
Hi, I can change it.. I developed this habbit of putting the constant first so that if you forget the second = in == it wont become an assignment, but results in a compiler error 😇

Reply:
😅